supabase/migrations/10_spec_ext.sql

```diff
@@ -76,12 +76,16 @@ comment on view combined_grants_ext is
 
 
 -- Extended view of live catalog specifications.
-create view live_specs_ext as
+CREATE VIEW live_specs_ext as
```

Comment:
nit: `create view` for consistency. There's a write-up in the very first migration on SQL coding style, at the top.